src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixLegacyResourceDesignerStep.cs

```diff
@@ -144,6 +148,7 @@ protected override void FixBody (MethodBody body, TypeDefinition designer)
 						instructions.Add (i, newIn);
 					} else {
 						LogMessage ($"DEBUG! Failed to find {key}!");
+						throw new InvalidOperationException (string.Format (CultureInfo.CurrentCulture, Resources.XA_Fixup_Designer_Error, key.Replace ("::", "/")));
```

Comment:
It becomes "a" MSbuild error... if its reasonable or not is something for @jonpryor  :P.
Its not coded, its a Linker Error.

My guess is if we want a coded error, we'd need to make some changes to the linker.

Reply:
I added the error messages we get to the commit. 
Unfortunately they are different between Debug and Release builds. This is because one is raised by our LinkAssembliesNoShrink task and one by the linker itself. There is no way that I can see to get a consistent error code for both.... unless we embed the code into the actual text or the error.

Reply:
I was hoping there was a way to have a consistent error code across both implementations. As its us raising the error.